projects/plugins/jetpack/extensions/blocks/donations/controls.js

```diff
@@ -21,31 +20,28 @@ import {
 	ToolbarItem,
 } from '@wordpress/components';
 import { __ } from '@wordpress/i18n';
+import { BlockControls, InspectorControls } from '@wordpress/block-editor';
 import { DOWN } from '@wordpress/keycodes';
 
+/**
+ * External dependencies
+ */
+import getSiteFragment from '../../shared/get-site-fragment';
```

Comment:
This is already imported on line 5

Reply:
Hmmm that is odd, the build fails with that code. I've seen that the External dependencies got duplicated in [cc280c2765887da05770fed063331360f7c938d6](https://github.com/Automattic/jetpack/pull/22710/commits/cc280c2765887da05770fed063331360f7c938d6). I'll remove it

Reply:
- [ ] Remove duplicated External dependencies.